src/main/java/lotto/domain/lotto/Lotto.java

```diff
@@ -0,0 +1,65 @@
+package lotto.domain.lotto;
+
+import lotto.exception.LottoNumDuplicateException;
+import lotto.exception.LottoNumRangeException;
+import lotto.exception.LottoSizeException;
+
+import java.util.HashSet;
+import java.util.List;
+import java.util.Set;
+
+public class Lotto {
+    private final List<Integer> numbers;
+
+    public Lotto(List<Integer> numbers) {
+        validate(numbers);
+        validateNumRange(numbers);
+        validateDuplicate(numbers);
+        this.numbers = numbers;
+    }
+
+    private void validate(List<Integer> numbers) {
+        if (numbers.size() != 6) {
+            throw new LottoSizeException();
+        }
+    }
+
+    // TODO: 추가 기능 구현
+    private void validateDuplicate(List<Integer> numbers) {
+        Set<Integer> hashNumbers = new HashSet<>();
+        for(Integer number : numbers) {
+            hashNumbers.add(number);
+        }
```

Comment:
(추천드려요!)
```suggestion
        Set<Integer> hashNumbers = new HashSet<>(numbers);
```
이렇게 변경 가능합니다!. 추가로 이방법 말고도 addAll을 활용한다면 좋을 것 같아요 :)